tests/test_persistence.py

```diff
@@ -52,3 +52,44 @@ def test_libsecret_persistence(temp_location):
         {"my_attr_1": "foo", "my_attr_2": "bar"},
         ))
 
+@pytest.mark.skipif(
+    is_running_on_travis_ci, reason="Requires manual testing")
+def test_windows_empty_file_exists_before_first_use():
+    test_folder = tempfile.mkdtemp(prefix="msal_extension_test_windows_token_cache_roundtrip")
+    cache_file = os.path.join(test_folder, 'msal.cache')
+    open(cache_file, 'w+')
```

Comment:
```suggestion
    open(cache_file, 'w')
```
By the way, this conceptually also be inside the try...finally... block, unless we switch to use the `temp_location` approach, as suggested above.